src/etc/cargo.bashcomp.sh

```diff
@@ -120,7 +120,7 @@ _cargo()
 } &&
 complete -F _cargo cargo
 
-__cargo_commands=$(cargo --list 2>/dev/null | tail -n +2)
+__cargo_commands=$(cargo --list 2>/dev/null | tail -n +2 | awk 'NR>1 {print $1}')
```

Comment:
The `tail` is not necessary with the `NR` in awk.  Indeed, this is now skipping the first command (bench).